src/views/Proposals.vue

```diff
@@ -27,14 +27,21 @@
     </Container>
     <Container :slim="true">
       <Block :slim="true">
-        <div class="px-4 py-3 bg-gray-dark">
+        <div class="px-4 py-3 bg-gray-dark menu-tabs">
```

Comment:
@emiex Can you add these change for mobile in another PR? Also you can use directly primercss classes "overflow-auto white-space" no need to create specific classes for the tab.

Reply:
can but also was planning on doing some custom design for smaller screens, for now we can use `overflow-auto white-space`